account.py

```diff
@@ -0,0 +1,9 @@
+class Account:
+
+    def __init__(self,id,email,access_token,refresh_token):
+        self.id = id
+        self.email = email
+        self.access_token = access_token
+        self.refresh_token = refresh_token
+    
```

Comment:
Yes this is the model for Accounts in user service. This will not be shared with the other services but only within the user service.

The design pattern that I am aming for is called **M**odel-**V**iew-**C**ontroller or MVC for short.

This will make more sens further down when the user service will grow in complexity.